mkdocs.yml

```diff
@@ -22,17 +22,19 @@ nav:
   - Server:
       - Configuration: server/configuration.md
       - Auth: server/auth.md
+      - Google Auth: server/google-auth.md
   - Integrations:
       - Apache Spark: integrations/unity-catalog-spark.md  
       - Daft: integrations/unity-catalog-daft.md
       - DuckDB: integrations/unity-catalog-duckdb.md
       - PuppyGraph: integrations/unity-catalog-puppygraph.md
       - SpiceAI: integrations/unity-catalog-spiceai.md
       - XTable: integrations/unity-catalog-xtable.md
+      - Unity Catalog Celerdata: integrations/unity-catalog-celerdata.md
   - Deployment: deployment.md
 
-exclude_docs: |
```

Comment:
@Brijeshthummar02 Please leave as is. This definitely works and with the update the doc test is failing again.